src/components/playback/playbackmanager.js

```diff
@@ -1409,14 +1409,23 @@ class PlaybackManager {
 
             const apiClient = ServerConnections.getApiClient(self.currentItem(player).ServerId);
 
-            apiClient.getEndpointInfo().then(function (endpointInfo) {
+            Promise.all([
+                apiClient.getEndpointInfo(),
+                apiClient.getCurrentUser()
+            ]).then(([endpointInfo, user]) => {
                 const playerData = getPlayerData(player);
                 const mediaType = playerData.streamInfo ? playerData.streamInfo.mediaType : null;
 
                 let promise;
                 if (options.enableAutomaticBitrateDetection) {
                     appSettings.enableAutomaticBitrateDetection(endpointInfo.IsInNetwork, mediaType, true);
                     promise = apiClient.detectBitrate(true);
+                } else if (user?.Policy
+                    && ((mediaType === 'Video' && !user.Policy.EnableVideoPlaybackTranscoding)
+                        || (mediaType === 'Audio' && !user.Policy.EnableAudioPlaybackTranscoding))) {
+                    // If transcoding is not allowed, we cannot use the specified bitrate - force bitrate detection
+                    console.warn('Transcoding is not allowed, cannot use the specified bitrate - force bitrate detection');
+                    promise = apiClient.detectBitrate(true);
```

Comment:
We could add `appSettings.enableAutomaticBitrateDetection(endpointInfo.IsInNetwork, mediaType, true);` to stop warning next play.